ckan/views/dataset.py

```diff
@@ -754,6 +754,12 @@ def get(
         pkg = context.get(u"package")
         resources_json = h.json.dumps(data.get(u'resources', []))
 
+        # if dataset was created via API, license_id is None
+        # set it to nonspecified to avoid accidental update
```

Comment:
How it prevents accidental updates? From my point of view:
1. > dataset was created via API, license_id is None
2. Edit form show, `license_id` set to `notspecified`
3. User DOESN'T change anything and clicks **Save**.
4. `notspecidied` stored inside DB as new license value <- **ACCIDENTAL UPDATE**